tests/kafkatest/services/console_consumer.py

```diff
@@ -132,6 +137,8 @@ def start_cmd(self):
         cmd += " /opt/kafka/bin/kafka-console-consumer.sh --topic %(topic)s --zookeeper %(zk_connect)s" \
             " --consumer.config %(config_file)s" % args
 
+        if self.new_consumer:
+            cmd += " --new-consumer --bootstrap-server %s"  % self.kafka.bootstrap_servers(self.security_protocol)
```

Comment:
Since we're adding support for the new consumer instead of assuming old consumer, should we move the `--zookeeper` option into an else clause here?